pages/[slug].js

```diff
@@ -128,7 +135,7 @@ export default function Post({ postData, status, categoryPosts, mixpanel }) {
         var element = contentRef.current.querySelector(linkHref);
         if (element) {
           window.scrollTo({
-            top: element.offsetTop - 90,
+            top: element.offsetTop - 110,
```

Comment:
110 will be the height of the header. Let's set the header's height here instead of manual value